handlers/http.go

```diff
@@ -47,6 +64,9 @@ func NewHttpHandler(options *HttpHandlerOptions) Handler {
 		Method(options.Method).
 		Use(timeout.Request(2 * time.Minute))
 
+
+	baseUrl, _ := url.Parse(endpoint)
```

Comment:
This doesn't compile there's no `endpoint` variable

Reply:
fixed